charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,8 +18,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.3.0
-appVersion: 0.43.2
+version: 12.4.1
+appVersion: 0.44.0
```

Comment:
Please do not change the appVersion. This corresponds to the mage version [here](https://github.com/prometheus-community/helm-charts/blob/64e66a78a379eed1b80fb1e8633838de6647d53f/charts/kube-prometheus-stack/values.yaml#L1416)